proposals/dao/fip_38.ts

```diff
@@ -0,0 +1,118 @@
+import { ethers } from 'hardhat';
+import chai, { expect } from 'chai';
+import CBN from 'chai-bn';
+import {
+  DeployUpgradeFunc,
+  NamedContracts,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '../../types/types';
+import { TransactionResponse } from '@ethersproject/providers';
+import { expectApprox } from '@test/helpers';
+
+chai.use(CBN(ethers.BigNumber));
+
+// Constants
+const TOKEMAK_REWARDS_ADDRESS = '0x79dD22579112d8a5F7347c5ED7E609e60da713C5';
+const TOKEMAK_WETH_POOL_ADDRESS = '0xD3D13a578a53685B4ac36A1Bab31912D2B2A2F36';
+const TOKEMAK_TOKE_POOL_ADDRESS = '0xa760e26aA76747020171fCF8BdA108dFdE8Eb930';
```

Comment:
These should probably be added to mainnetAddresses.json right?

Reply:
I just needed them here, but I can add them. What `artifactName` do I use ?